product_to_scale_bizerba_extended/models/product_scale_group.py

```diff
@@ -19,14 +19,14 @@ class product_scale_group(models.Model):
     def send_all_to_scale_create(self):
         for scale_group in self:
             (scale_group.product_ids
-             .filtered(lambda r: r.to_weight and r.sale_ok)
+             .filtered(lambda r: r.is_in_scale())
```

Comment:
This module doesn't depends on the pos module, so we should no use the `to_weight` attribute.